README.md

```diff
@@ -101,7 +101,33 @@ would in a Django shell.
 
 Unlike our other web applications this project is not using tox. To run tests locally, run this command instead:
 
+    ### PYTHON TESTS/LINTING
+    # Run Python tests
     docker-compose run --rm web pytest
+    # Run Python tests in a single file
+    docker-compose run --rm web pytest /path/to/test.py
+    # Run Python test cases in a single file that match some function/class name
+    docker-compose run --rm web pytest /path/to/test.py -- -k test_some_logic
+    # Run Python linter
+    docker-compose run --rm web pylint
+    
+    ### JS/CSS TESTS/LINTING
+    # We also include a helper script to execute JS tests in most of our projects 
+    # (this file may exist at the project root or in ./scripts/test)
+    docker-compose run --rm watch ./scripts/test/js_test.sh
+    # Run JS tests in specific file
+    docker-compose run --rm watch ./scripts/test/js_test.sh path/to/file.js
+    # Run JS tests in specific file with a description that matches some text
+    docker-compose run --rm watch ./scripts/test/js_test.sh path/to/file.js "should test basic arithmetic"
+    # Run the JS linter
+    docker-compose run --rm watch npm run lint
+    # Run SCSS linter
+    docker-compose run --rm watch npm run scss_lint
+    # Run the Flow type checker
+    docker-compose run --rm watch npm run-script flow
```

Comment:
need to change it ? @ahmed-arbisoft

Reply:
@asadiqbal08 not exactly. I just checked the documentation and `run-script` will work fine here as well. Please ignore my last comment on this.

Reply:
its working on my system.